docs/_build/html/_sources/index.md.txt

```diff
@@ -22,59 +22,66 @@ or logs right in your terminal.
 
 ## Install Tilt
 
-Download the Tilt binary on
-[the github releases page](https://github.com/windmilleng/tilt/releases).
+Download the latest Tilt release from
+[GitHub](https://github.com/windmilleng/tilt/releases). Read the [Installation Guide](install.html) for details and prerequistes.
 
-Tilt expects that you already have Docker and `kubectl` installed.
-Read the more detailed [Installation Guide](install.html)
-to help you `tilt up` quickly.
+## Describe Your Workflow
 
-## Configure Your Workflow to Share With Your Team
+A Tiltfile is a program that connects your existing Docker and Kubernetes configurations:
 
-Down with YAML!
+```python
+# Example Tiltfile for a k8s app with two microservices
 
-Configure Tilt with a `Tiltfile`, written in a small subset of Python called
-[Starlark](https://github.com/bazelbuild/starlark#tour).
+# Deploy: tell Tilt what yaml to apply
+k8s_yaml('app.yaml')
 
-To get started, check out some [examples](first_example.html) or dive into the
-[API reference](api.html).
+# Build: tell Tilt what images to build from which directories
+docker_build('companyname/frontend', 'frontend')
+docker_build('companyname/backend', 'backend')
+```
 
-## Community
+Set up Tilt in 15 minutes with the [Tutorial](tutorial.html).
+
+## See More
```

Comment:
this reads kiiiinda like a section that's about to link to some additional resources, like "see also" or "read more"? 🤷‍♀️

Reply:
Yes, I also felt uneasy but couldn't come up with anything better.